src/main/java/util/miscellaneous/DukeOperation.java

```diff
@@ -0,0 +1,321 @@
+package util.miscellaneous;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import util.task.Deadline;
+import util.task.Event;
+import util.task.Task;
+import util.exception.NoDateException;
+import util.exception.NoTaskException;
+import util.exception.NoItemException;
+import util.task.Todo;
+
+public class DukeOperation implements Chatbot {
+    public static void linePrinter() {
+        System.out.print("\t");
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void printGreeting() {
+        linePrinter();
+        System.out.println("\t" + GREETING_MSG_01);
+        System.out.println("\t" + GREETING_MSG_02);
+        linePrinter();
+
```

Comment:
There are blank lines like these before the closed curly brackets, yet other methods do not have this empty line. You could make the layout more consistent.